dask/bytes/core.py

```diff
@@ -141,10 +141,13 @@ def write_bytes(data, urlpath, name_function=None, compression=None,
         raise NotImplementedError("Unknown protocol for writing %s (%s)" %
                                   (protocol, urlpath))
 
-    files = open_files_write(paths, **storage_options)
-    out = [delayed(write_block_to_file)(v, f, compression, encoding)
-           for (v, f) in zip(data, files)]
-    return out
+    keys = ['write-block-%s' % path for path in paths]
```

Comment:
Since this depends only on path, it seems natural to have it part of the key - but path can be rather long. Should it be tokenized anyway?